yml/dp02_patches.yaml

```diff
@@ -0,0 +1,58 @@
+---
+name: dp02_dc2_images
```

Comment:
I'd hope we wouldn't need a separate database just for one table?

Reply:
That was just temporary because I was worried about unexpected consequences if I just added it to the existing file on the fly.  @fritzm reassured me about that on Slack earlier.